generators/app/templates/ext-wizard/src/wizards/db-ops/wizard/pages/page2.ts

```diff
@@ -0,0 +1,88 @@
+import * as azdata from 'azdata'
+import * as vscode from 'vscode'
+import {ConnectionModel} from '../api/models'
+import {BasePage} from '../api/basePage'
+
+/**
+ * Represents the second page of the Wizard, which allows the user to select
+ * one of several operations on their selected connection
+ */
+export class Page2 extends BasePage {
+    private tasks : Map<string, any>;
+    private optionsDropdown: azdata.DropDownComponent | undefined;
+    private wizard: azdata.window.Wizard;
+
+    public constructor(wizardPage: azdata.window.WizardPage, model: ConnectionModel,
+            view: azdata.ModelView, width: number, wizard : azdata.window.Wizard) {
+        super(wizardPage, model, view, width);
+        this.tasks = new Map();
+        this.wizard = wizard;
+	}
+
+	async start(): Promise<boolean> {
+        this.tasks.set('New Query', () => this.newQuery());
+        this.tasks.set('Expand and Refresh Connection', () => this.refreshConnection());
+        this.tasks.set('View Connection String', () => this.viewConnectionString());
+
+        let optionsComponent = await this.createOptionsDropdown();
+
+        let layout = {componentWidth: this.width};
+        let formBuilder = this.view.modelBuilder.formContainer().withFormItems(
+            [optionsComponent], layout);
+
+        if (this.optionsDropdown) {
+            this.wizard.doneButton.onClick(() => this.tasks.get(String(this.optionsDropdown?.value)).call());
+        }
+
+        await this.view.initializeModel(formBuilder.component());
+        return true;
+    }
+
+    async onPageEnter(): Promise<boolean> {
+		return true;
+    }
+
+    private async createOptionsDropdown(): Promise<azdata.FormComponent> {
+		let optionsDropdown = this.view.modelBuilder.dropDown().withProperties({
+            required: true,
+            values: Array.from(this.tasks.keys())
+        }).component();
+
+        this.optionsDropdown = optionsDropdown;
+		return {
+			component: optionsDropdown,
+			title: 'Select a Task:'
+		};
+    }
+
+    // working with VS Code file APIs
+    private newQuery(){
+        let sqlContent = 'SELECT * FROM ' + this.model.database + '.INFORMATION_SCHEMA.TABLES';
```

Comment:
I thought we're creating separate sample to show how to work with query editor.

Reply:
Ah yes sorry, this was the work done previously. I included it to get your feedback, but based on our conversation, it probably makes sense for me to remove this template.